### ISSUE TITLE
[Bug] New validation screen doesn't adjust Severity  in issued CVE

### ISSUE THREAD

**Issue by jaapmarcus** [418sec/huntr#2203]
For my repository I validated yesterday a venerability in Huntr.dev:

When I went to adjust Severity I had the option in a select box to select: None, Low, Medium, High, Critical

When selecting "Severity" to low the form closed and saved the results. 

After the vulnerability has been patched / released @JamieSlome  published the CVE (https://nvd.nist.gov/vuln/detail/CVE-2022-0986)

<img width="1034" alt="Screenshot 2022-03-16 at 21 46 46" src="https://user-images.githubusercontent.com/9754650/158688019-dd042165-5175-44d9-8815-c97a72ed42e1.png">

And Huntr.dev
<img width="423" alt="Screenshot 2022-03-16 at 21 41 49" src="https://user-images.githubusercontent.com/9754650/158688067-7c95f661-169b-4bb4-9c38-8e2ba810effa.png">

Report:
https://huntr.dev/bounties/57635c78-303f-412f-b75a-623df9fa9edd/

After the vulnerability has been "adjusted" the the correct CVSS score should be listed in the CVE...

**Comment by JamieSlome** [comment]
@jaapmarcus - thanks for notifying us about the issue.

It looks like a gap in my manual process of assigning the CVE. Any reports that have been marked as `None` or `Low` will no longer receive CVEs (which is why this report didn't receive one). I should have first checked whether this was the case with this report before publishing the CVE manually.

I will keep this ticket referenced as it certainly demonstrates the need for better automation around CVE assignment and process (#2194).

In the meantime, I have adjusted the CVSS of the CVE to low 👍 (https://github.com/CVEProject/cvelist/pull/4952). Once this has been merged the CVE should update within the hour.
